ddp/ddp_pipeline/src/main/java/org/mskcc/cmo/ks/ddp/pipeline/util/DDPPatientListUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Memorial Sloan-Kettering Cancer Center.
+ * Copyright (c) 2018-2023 Memorial Sloan-Kettering Cancer Center.
```

Comment:
It's not clear whether we have standardized on a having spaces around the hyphen between the two years at the top of every file ... but I think that would be a good standard. (there is another no space instance in a later file)